analysis/topeft_run2/run_analysis.py

```diff
@@ -255,7 +255,10 @@ def LoadJsonToSampleName(jsonFile, prefix):
             'tasks_accum_log': 'tasks.log',
 
             'environment_file': remote_environment.get_environment(
-                extra_pip_local = {"topeft": ["topeft", "setup.py"]},
+                extra_pip_local = {
+                    "topeft": ["topeft", "setup.py"],
+                    "topcoffea": ["topcoffea", "setup.py"]
```

Comment:
Are these needed? The remove environment scripts are inside TopCoffea, and already watch the local pip install.
https://github.com/TopEFT/topcoffea/blob/31a6474c68e1685fe87f2eef94ced151beb881ff/topcoffea/modules/remote_environment.py#L44